src/astro/sql/operators/load_file.py

```diff
@@ -182,7 +190,7 @@ def replace_illegal_columns_chars(char: str, database: BaseDatabase) -> str:
 
 
 def load_file(
-    input_file: File,
+    input_file: Union[File, str],
```

Comment:
@kaxil @tatiana should we actually allow people to just pass a string in? I feel like forcing File objects will be a good way to teach people to write things "properly" where they can easily change systems later

Reply:
We have reverted this changes